examples/preprocessing/count_featurizer_transformer.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import print_function
+import matplotlib.pyplot as plt
+import numpy as np
+import time
+from sklearn.preprocessing.data import CountFeaturizer
+from collections import OrderedDict
+from sklearn.datasets import make_classification
+from sklearn.ensemble import RandomForestClassifier, ExtraTreesClassifier
+
+# with reference to auto_examples/ensemble/plot_ensemble_oob
+
+
+RANDOM_STATE = 123
+
+n_datapoints = 500  # 500
+n_informative = 10  # 15
+n_features = 10  # 25
+n_redundant = 0
+
+# Generate a binary classification dataset.
+X, y = make_classification(n_samples=n_datapoints, n_features=n_features,
+                           n_clusters_per_class=1, n_informative=n_informative,
+                           n_redundant=n_redundant, random_state=RANDOM_STATE)
+
+X = X * 10
+# only make these selected features "categorical"
+discretized_features = [0, 1]
+for feature in discretized_features:
+    X[:, feature] = (X[:, feature]).astype(int)
```

Comment:
How many unique levels are there? Also, you should compare against using ``OneHotEncoder`` on the same features.